src/teleop/gui/src/components/AutonTask.vue

```diff
@@ -72,10 +72,12 @@ export default {
 
             navBlink: false,
             greenHook: false,
+            ledColor: 'blue',
```

Comment:
Do we ever want to send 'off' for ledColor? Or will ESW only be turning the LED on during auton anyways?

Reply:
Yea you probably don't want to send 'off' unless there ever becomes a day where auton happens before another mission (in which case, the other guis can send a turn off kind of thing)

Reply:
Ok so this is all good then?